lighthouse-core/test/report/v2/report-generator-test.js

```diff
@@ -143,6 +146,20 @@ describe('ReportGeneratorV2', () => {
       assert.ok(result.includes('LIGHTHOUSE_JAVASCRIPT'), 'cannot be tricked');
     });
 
+    it('should inject the report templates', () => {
+      const page = jsdom.jsdom(new ReportGeneratorV2().generateReportHtml({}));
+      const templates = jsdom.jsdom(TEMPLATES_FILE);
+
+      assert.deepStrictEqual(page.querySelectorAll('template[id^="tmpl-"]'),
```

Comment:
> switched to using lengths

lulz, well the `assert.deepEqual` was checking that, but I guess at least now it's explicit